waflog/read.go

```diff
@@ -102,15 +102,22 @@ func (ll *FTWLogLines) CheckLogForMarker(stageID string) []byte {
 	for err == nil && len(line) == 0 {
 		line, _, err = scanner.LineBytes()
 	}
-	if err != nil {
-		if err == io.EOF {
-			return nil
+
+	// Try to find the marker in the last and the next to last
+	// line. Some errors in httpd will be printed **after** the
+	// ModSecurity output (e.g. 404 with filesystem lookup).
+	for i := 1; i >= 0; i-- {
```

Comment:
Can this be made configurable? For FTW tests with coraza-proxy-wasm, the logs are combined with Envoy logs - we [grep](https://github.com/corazawaf/coraza-proxy-wasm/blob/main/ftw/docker-compose.yml#L40) into a separate lot for FTW to separate them, but if the lookback could be configured to a high-ish number in those tests, we could probably get rid of that

Reply:
> Why not just search until EOF?

Because you do not want to look through 200MB 20 times :)

Yes, we can absolutely make the lookback configurable. But maybe we can use a different approach. If we knew, which log lines we could ignore, we wouldn't have to pick an arbitrary number. So how about we add a configuration option for a regular expression (or multiple) that can filter unwanted log lines? In your case, you'll already know the pattern.

Reply:
Thanks - I think a regex could work well for us. Though maybe we don't need it - is there any reason not to search until EOF or `X-CRS-Test`? It shouldn't result in ever going through an entire huge log file, and either the matched line will be for the current test, or the previous test and we'll retry and look again.